### DISCUSSION TITLE
YouTube + заблокированные сайты вместе

### DISCUSSION THREAD

**Discussion by YanusBeda** [bol-van/zapret#399] [General]
Подобрал стратегии дурения для : 
Youtube  "--dpi-desync=split2"
Для Блок.сайтов "--dpi-desync=fake --dpi-desync-ttl=7"
Не могу совместить оба эти параметра. Подскажите можно эти параметры совместить? У меня что то одно работает, вместе нет.

**Comment by numbereleven-a** [comment]
[bolvan - Я щас делаю мультистратегию. Будет типа такого.
](https://ntc.party/t/zapret-%D0%BE%D0%B1%D1%81%D1%83%D0%B6%D0%B4%D0%B5%D0%BD%D0%B8%D0%B5/726/615)

**Comment by Miles1727** [comment]
Такая же проблема правила вместе не работают

**Reply by BubaMuba** [reply]
Работают. Обновитесь и поправьте конфиг.

**Comment by YanusBeda** [comment]
Обновил Zapret. Не работает вместе. В описании про мультистратегию написано ,что разделять параметром --new . Пробовал 
"--dpi-desync=split2 --new --dpi-desync=fake --dpi-desync-ttl=7" - работает только первая часть до --new. Может кто привести пример? Непонятно как подвязывать для стратегий хостлисты, они должны разные быть и ли как раньше один? Еще в конфиге появился пункт "NFQWS_OPT_DESYNC_HTTPS_SUFFIX=" для чего?

**Reply by bol-van** [reply]
В первой части до --new нет фильтра, поэтому этот профиль всегда и выбирается.

https://ntc.party/t/zapret-whats-new/61/149

**Comment by YanusBeda** [comment]
Спасибо за помощь и разработку!! Все заработало. Даже на телике Samsung завел, нужно было youtube.com внести в обычный список хостов (zapret-hosts-user.txt) , иначе программа просто на запуске черный экран Малевича выдает. Почему так ХЗ , на андроидах и ПК и так и так работает.

**Reply by maxbur89** [reply]
С какими параметрами удалось запустить?

**Reply by YanusBeda** [reply]
> С какими параметрами удалось запустить?
"--hostlist=/opt/Youtube.txt --dpi-desync=split2 --new --dpi-desync=fake --dpi-desync-ttl=7"  провайдерТТК
В файл Youtube.txt внести все хосты для ютюба ,работает стратегия "--dpi-desync=split2" 
 В файл zapret-hosts-user.txt заносить другие хосты для которых работает стратегия "--dpi-desync=fake --dpi-desync-ttl=7"
На ПК и Андроидах работает. 
Что бы работало на теликах Samsung пришлось "youtube.com" убрать из "Youtube.txt" и перенести в "zapret-hosts-user.txt".

**Comment by cicsaw** [comment]
youtube.txt вы резолвите или просто доменами в opt кладёте?

**Reply by YanusBeda** [reply]
> youtube.txt вы резолвите или просто доменами в opt кладёте?

Просто доменами , типа "googlevideo.com"

**Comment by YanusBeda** [comment]
Так же рекомендую установить на роутер DNSCrypt2 [https://habr.com/ru/articles/556536/](url) , с ним пробивает больше хостов
